packages/components/Menu/src/MenuPopover/useMenuPopover.ts

```diff
@@ -91,6 +92,10 @@ export const useMenuPopover = (props: MenuPopoverProps): MenuPopoverState => {
     };
   });
 
+  const vibrancyViewProps: VibrancyViewProps = {
```

Comment:
These changes could have been in their own PR separate from implementing VibrancyView

Reply:
I can factor out any changes to Callout and menu I think and keep it simpler